connectors/xai_connector.py

```
@@ -20,25 +20,25 @@
     Production-ready connector for xAI's Grok models with MCP integration.
-    
+
     Features:
     - Grok-3 model access
     - Live search capabilities
     - Streaming support
     - Material Design 3 compliant responses
     """
-    
+
     def __init__(self, api_key: Optional[str] = None):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)